### DISCUSSION TITLE
How can i scan bluetooth device on Ios in android startDiscovery() works  but in ios i get error Method not implemented

### DISCUSSION THREAD

**Discussion by janakbastakoti** [kenjdavidson/react-native-bluetooth-classic#251] [Q&A] [answered]
let unpaired = await RNBluetoothClassic.startDiscovery();

**Comment by kenjdavidson** [comment, marked as answer]
You can't. It's not supported. See the documentation.  Ios and mfi is very limited, this is just the way it is. Unless they've changed it, which I doubt.

**Comment by janakbastakoti** [comment]
Hello sir is there is any way to connect my react native ios app and raspberry pi through bluetooth  and transfer data between app and raspberry pi

**Comment by kenjdavidson** [comment]
You need to have the mfi protocols for the raspberry pi. I don't know if those exist. You'll have to google around, but unlikely.

**Reply by janakbastakoti** [reply]
Thank you sir
